schemas/doi.json

```diff
@@ -0,0 +1,43 @@
+{
+  "title": "DOI",
+  "description": "Persistent identifier",
```

Comment:
The Digital Object Identifier (DOI) is a persistent ID assigned to published items.